### PR TITLE
refactor: addons to hook

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

**Comment by ykhedher** [comment]
[refactor addons to hooks](https://trello.com/c/wAfayVqR/825-refactor-addons-to-hooks)

**Comment by ykhedher** [comment]
[Refactor redux - Addons](https://trello.com/c/CspOKbaZ/816-refactor-redux-addons)

File: src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.jsx [hunk 1]
```diff
@@ -17,15 +17,51 @@ import {
...
+        try {
+            await updateAddon({ ...addon, enabled: !addon.enabled });
+            refetchAddons();
+            setToastData({
+                type: 'success',
+                title: 'Success',
+                text: 'Addon state switched successfully',
+            });
+        } catch (e) {
+            setToastData({
```
Comment (FredrikOseberg) [inline src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.jsx:39]: You can use setToastApiError here, which will save you having to specify the type

File: src/component/addons/AddonList/AvailableAddons/AvailableAddons.jsx [hunk 2]
```diff
@@ -13,6 +13,8 @@ import { CREATE_ADDON } from '../../../providers/AccessProvider/permissions';
 import PropTypes from 'prop-types';
 
 const AvailableAddons = ({ providers, getIcon, hasAccess, history }) => {
+    
+    console.log('this is ', providers)
```
Comment (FredrikOseberg) [inline src/component/addons/AddonList/AvailableAddons/AvailableAddons.jsx:17]: Remove this.

**Pull request merged**